src/CMakeLists.txt

```diff
@@ -143,6 +143,7 @@ endif()
 # Commands to build initial module objects.
 file(MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.build")
 set (CXX_WARNING_FLAGS -Wall -Werror -Wno-unused-function -Wcast-qual)
+set (RUNTIME_CXX_FLAGS -O3 -fno-vectorize -ffreestanding -fno-blocks -fno-exceptions -fno-unwind-tables -fpic -std=gnu++98)
```

Comment:
Why did you remove `-fno-ms-compatibility`?

Reply:
Like the commit message says, I want to keep the same flags for both Makefile and CMake. See https://github.com/halide/Halide/blob/aa33d0a/Makefile#L1039 there is not `-fno-ms-compatibility`. We can add it in both if you prefer, but we need to keep them in sync.

Reply:
Fair enough -- I don't recall offhand why that flag is here in the first place, and all the tests are passing.